graylog2-server/src/main/java/org/graylog2/shared/journal/Journal.java

```diff
@@ -31,6 +31,10 @@ public interface Journal {
 
     void flush();
 
+    default double getJournalUtilization() {
```

Comment:
How do we signal that a `Journal` implementation doesn't support reporting its utilization? Using `0` here as a default value would be misleading because that's a valid value for an empty journal.

Either we use something like `-1` or let the method return an `Optional<Double>` to signal that a journal doesn't support getting its utilization.

We should also add Javadoc to explain the returned value.